src/mumble/MainWindow.cpp

```diff
@@ -3811,3 +3813,24 @@ void MainWindow::destroyUserInformation() {
 		}
 	}
 }
+
+void MainWindow::on_muteCuePopup_triggered() {
+	if (Global::get().s.muteCueShown || Global::get().inConfigUI) {
```

Comment:
Why do we explicitly exclude the case where the user is currently in the settings? I would assume that you could just as well be surprised by the mute cue in the settings as you could while using Mumble normally :thinking:

Reply:
Hi, good observation.  This was kind of a subjective decision.  During testing I noticed that when opening the settings UI, the mute cue would always activate so the pop up would show.  I wasn't sure if this was just an issue in my own environment but I also just kind of felt like it would be less overwhelming for the user if there wasn't a popup.  Nonetheless, I can change it if that seems like the better decision.

Reply:
I am in favor of not showing the popup while the user is browsing the settings. The Mumble settings are somewhat overwhelming as is, and adding another popup which may or may not come very surprisingly will just lead to users dismissing the popup entirely. I bet a not insignificant amount of users would not even notice the audible mute cue, when they are focusing on the text in the settings ;D

Reply:
> his was kind of a subjective decision. During testing I noticed that when opening the settings UI, the mute cue would always activate so the pop up would show. I wasn't sure if this was just an issue in my own environment but I also just kind of felt like it would be less overwhelming for the user if there wasn't a popup.

Ah yes. When you open (or apply) the settings, the audio system is reset which leads to a short input-triggering that'll be picked up by the mute cue logic.

> I bet a not insignificant amount of users would not even notice the audible mute cue, when they are focusing on the text in the settings ;D

There might be some truth in that xD

----

Alright, then let's keep it as it is :+1: